phpstan.neon.dist

```diff
@@ -1,5 +1,8 @@
+includes:
+	- phpstan-baseline.neon
+
 parameters:
-    level: 2
+    level: 5
```

Comment:
hab das level erhöht sodass wir qualitativ bessere fehlermeldungen bekommen und alle vorhandenen fehler der codebase in die baseline generiert mittels `composer phpstan-baseline`